ige/src/plugin/audio/AudioEngine.cpp

```diff
@@ -32,8 +33,23 @@ AudioEngine::AudioEngine(const char* deviceName)
     AudioEngine::get_native_exception();
 }
 
+AudioEngine::AudioEngine(AudioEngine&& engine)
+{
+    *this = std::move(engine);
```

Comment:
using the `std::move` function template requires including the `<utility>` standard header